plugin/src/main/kotlin/tanvd/grazi/JreProxySelector.kt

```diff
@@ -0,0 +1,80 @@
+package tanvd.grazi
+
+import org.eclipse.aether.repository.*
+import org.eclipse.aether.repository.Proxy
+import org.eclipse.aether.repository.ProxySelector
+import java.net.*
+import java.util.*
+
+class JreProxySelector : ProxySelector {
```

Comment:
Wow. What is this?

Reply:
Its proxyselector for https

Reply:
Have we written on ourselves or it is taken from somewhere?

Reply:
Taken from Idea Community and rewritten to kotlin

Reply:
Ok, please add a comment about it (from where it taken)

Reply:
Done